rust/chains/abacus-ethereum/src/retrying.rs

```diff
@@ -185,74 +180,19 @@ impl JsonRpcClient for RetryingProvider<Http> {
                     warn!(attempt, next_backoff_ms, error = %e, "JsonRpcError in http provider; not retrying.");
                     HandleMethod::Halt(HttpClientError::JsonRpcError(e))
                 } else {
-                    info!(attempt, next_backoff_ms, error = %e, "JsonRpcError in http provider.");
+                    warn!(attempt, next_backoff_ms, error = %e, "JsonRpcError in http provider.");
                     HandleMethod::Retry(HttpClientError::JsonRpcError(e))
                 }
             }
             Err(HttpClientError::SerdeJson { err, text }) => {
-                info!(attempt, next_backoff_ms, error = %err, text = text,  "SerdeJson error in http provider");
+                warn!(attempt, next_backoff_ms, error = %err, text = text,  "SerdeJson error in http provider");
                 HandleMethod::Retry(HttpClientError::SerdeJson { err, text })
             }
         })
         .await
     }
 }
 
-#[async_trait]
-impl<C> JsonRpcClient for RetryingProvider<QuorumProvider<C>>
```

Comment:
why remove this?

Reply:
Dead code that is no longer used; just happened to notice it since I was mucking around this file. We decided that instead of doing
`RetryingProvider<QuorumProvider<HttpProvider>>` we would do `QuorumProvider<RetryingProvider<HttpProvider>>`.